addons/web/static/src/scss/form_view.scss

```diff
@@ -378,8 +378,9 @@
         clear: both; // For the notebook not to have alongside floating elements
         margin-top: $o-form-spacing-unit * 2;
 
-        .nav-tabs > .tab-pane {
+        .tab-content > .tab-pane {
             min-height: 100px;
```

Comment:
Maybe remove the min-height if there were none in enterprise before ?

Reply:
It was probably supposed to be in Enterprise before, but it was not applied because the selector was wrong ?

Reply:
That's why you can remove it :wink: At worst it was needed and we will put it back later.

Reply:
ok. I have updated the branch :)